pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -163,8 +163,19 @@ func (cc *cadvisorClient) MachineInfo() (*cadvisorapi.MachineInfo, error) {
 	return cc.GetMachineInfo()
 }
 
-func (cc *cadvisorClient) DockerImagesFsInfo() (cadvisorapiv2.FsInfo, error) {
-	return cc.getFsInfo(cadvisorfs.LabelDockerImages)
+func (cc *cadvisorClient) ImagesFsInfo(runtime string) (cadvisorapiv2.FsInfo, error) {
```

Comment:
@vishh So do we want to rely on cadvisor to provide image fs in the future info or not ?